src/server/terrain/biomes.zig

```diff
@@ -153,12 +222,23 @@ pub const Biome = struct {
 			}
 		}
 		self.vegetationModels = main.globalAllocator.dupe(StructureModel, vegetation.items);
+
+		const stripes = json.getChild("stripes");
+		var stripeList = main.ListUnmanaged(Stripe){};
+		defer stripeList.deinit(main.stackAllocator);
+		for (stripes.toSlice()) |elem| {
+			if (Stripe.initStripe(elem)) |stripe| {
+				stripeList.append(main.stackAllocator, stripe);
+			}
+		}
+		self.stripes = main.globalAllocator.dupe(Stripe, stripeList.items);
```

Comment:
Instead of using a list, you can intialize the slice directly with `stripes.toSlice().len`
This is possible since Stripe.initStripe() never returns null.